src/reanimated2/core.ts

```diff
@@ -213,6 +213,22 @@ if (!isWeb() && isConfigured()) {
       info: runOnJS(capturableConsole.info),
     };
     _setGlobalConsole(console);
+
+    const nativeRequestAnimationFrame = global.requestAnimationFrame;
+    let callbacks: Array<(time: number) => void> = [];
+    global.requestAnimationFrame = (
+      callback: (time: number) => void
```

Comment:
For the sake of consistency
```suggestion
      callback: (timestamp: number) => void
```